components/shared/toggle-visibility-input.tsx

```diff
@@ -1,19 +1,36 @@
-import { EyeIcon, EyeOffIcon } from "lucide-react";
-import { useState } from "react";
+import { EyeIcon, EyeOffIcon, Clipboard } from "lucide-react";
+import { useRef, useState } from "react";
 import { Button } from "../ui/button";
 import { Input, type InputProps } from "../ui/input";
+import { toast } from "sonner";
 
 export const ToggleVisibilityInput = ({ ...props }: InputProps) => {
 	const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+	const inputRef = useRef<HTMLInputElement>(null);
 
 	const togglePasswordVisibility = () => {
 		setIsPasswordVisible((prevVisibility) => !prevVisibility);
 	};
 
+	const copyToClipboard = () => {
+		if (!inputRef.current) return;
+
+		const inputElement = inputRef.current;
+		const text = inputElement.value;
+
+		inputElement.select();
+		navigator.clipboard.writeText(text);
+
+		toast.success("Value is copied to clipboard");
+	};
+
```

Comment:
We use a `copy-to-clipboard` package, we can remove this function and use instead:

`import copy from "copy-to-clipboard";`

Then 
```ts
copy("String");
```

Reply:
Yes. That'd be good. I have updated the PR.